### ISSUE TITLE
[Content Patcher] Split-Screen Bug

### ISSUE THREAD

**Issue by ellipszist** [Pathoschild/LookupAnything#914]
**Describe the bug**
If changing to custom language or German This makes it impossible to play local co-op.

**To Reproduce**
1. Install [Thai mod](https://www.nexusmods.com/stardewvalley/mods/7052) w/ requirements.
2. Create a save for local co-op
3. Press the 'Start' button to start local co-op.
4. Error occurs.

**Log file**
SMAPI-crash.txt
https://smapi.io/log/d7e04ae992cf46c0a0a08b192ef6b9cc
SMAPI-latest.txt.
https://smapi.io/log/3526bc0a47b844fb89f1c8f2ca017756

**Screenshots**
![image](https://github.com/Pathoschild/StardewMods/assets/43781364/b13e23ce-9441-4a63-b562-84853043abdb)

**Research**
Basically, if you have these xnb files in the game's Content folder, you will be able to play local co-op.
- Data\Achievements.th.xnb
- Data\Boots.th.xnb
- Data\CraftingRecipes.th.xnb
- Data\hats.th.xnb
- Data\NPCGiftTastes.th.xnb
- Fonts\SmallFont.th.xnb
- Fonts\SpriteFont1.th.xnb
- Strings\StringsFromCSFiles.th.xnb

**Comment by Pathoschild** [comment]
Fixed in the upcoming SMAPI 3.0.8. Thanks for reporting it!
